apolloschurchapp/src/ui/HomeTabHeaderFeatureConnected/HomeTabHeader.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { View } from 'react-native';
+import { useQuery } from '@apollo/client';
+import { H4, H3, styled } from '@apollosproject/ui-kit';
+import { format, startOfToday } from 'date-fns';
+
+import GET_USER_NAME from './getUserName';
+
+const StyledCard = styled(({ theme }) => ({
+  backgroundColor: theme.colors.paper || undefined,
+  ...{
+    paddingHorizontal: theme.sizing.baseUnit,
+    paddingVertical: theme.sizing.baseUnit * 0.75,
+  },
+}))(View);
+
+const DateText = styled(({ theme }) => ({
+  color: theme.colors.text.secondary,
+}))(H4);
+
+const GreetingText = styled(({ theme }) => ({
+  color: theme.colors.text.primary,
+}))(H3);
+
+const HomeTabHeader = () => {
+  const { data } = useQuery(GET_USER_NAME, {
```

Comment:
throw the query right here and just consolidate these all into one file that lives in the `ui` folder